docs/helping.md

```diff
@@ -74,8 +94,10 @@ Note that `close_delay` is a number, meaning the time in hours before the ticket
 Using "ac" as described later lets other people know how long until the ticket will close. "ac" meaning **A**uto **C**lose.
```

Comment:
Maybe mention that it's cause ticket bot doesn't mention the time a ticket was set to autoclose at. ^^